src/pudl/transform/params/ferc1.py

```diff
@@ -1692,6 +1695,21 @@
             "opex_per_kwh": PERKWH_TO_PERMWH,
             "net_generation_kwh": KWH_TO_MWH,
         },
+        "remove_invalid_rows": {
+            "invalid_values": [0, pd.NA, np.nan],
```

Comment:
Just a comment / thought, but now looking at this and the need for an import I see this means we'll be unable to store these parameters as JSON on disk.